arlo_server/routes.py

```diff
@@ -440,11 +438,15 @@ def update_jurisdictions_file(election_id=None):
     JURISDICTION_NAME = "Jurisdiction"
     ADMIN_EMAIL = "Admin Email"
 
-    missing_fields = [
-        field
-        for field in [JURISDICTION_NAME, ADMIN_EMAIL]
-        if field not in jurisdictions_csv.fieldnames
-    ]
+    missing_fields = (
+        [
+            field
+            for field in [JURISDICTION_NAME, ADMIN_EMAIL]
+            if field not in jurisdictions_csv.fieldnames
+        ]
+        if jurisdictions_csv.fieldnames
+        else [JURISDICTION_NAME, ADMIN_EMAIL]
+    )
```

Comment:
I'm not sure why this change was required, but `mypy` all of a sudden started complaining:

```
arlo_server/routes.py:444: error: Unsupported right operand type for in ("Optional[Sequence[str]]")
```

I infer that `jurisdictions_csv.fieldnames` is `Optional[Sequence[str]]` and that `in` does not apply to `None`. But I don't understand why this is only now a problem since the type of this shouldn't have changed.

Reply:
So weird! Here's an idea for something less verbose: 
```python
     missing_fields = [
        field
        for field in [JURISDICTION_NAME, ADMIN_EMAIL]
        if field not in (jurisdictions_csv.fieldnames or [])
    ]
```